pkg/service/backup_backend_local.go

```diff
@@ -111,7 +111,9 @@ func (local *BackupBackendLocal) FullBackupList() ([]model.BackupDetails, error)
 	for _, e := range entries {
 		if e.IsDir() {
 			details := toBackupDetails(e, backupFolder)
-			if details.LastModified.Before(lastRun) {
+			if details.LastModified.Before(lastRun) &&
```

Comment:
In the  RemoveFiles branch, when we have only one backup maximum, probably should filter it too, to be consistent.